src/main/java/com/june/propulsive/ItemGroups.java

```diff
@@ -0,0 +1,24 @@
+package com.june.propulsive;
+
+import net.fabricmc.fabric.api.itemgroup.v1.FabricItemGroup;
+import net.minecraft.item.ItemGroup;
+import net.minecraft.item.ItemStack;
+import net.minecraft.registry.Registries;
+import net.minecraft.registry.Registry;
+import net.minecraft.text.Text;
+import net.minecraft.util.Identifier;
+
+public class ItemGroups {
+    public static final ItemGroup PROPULSIVE_GROUP = Registry.register(Registries.ITEM_GROUP,
```

Comment:
We'll only have one item group, so move this into Items instead of its own class